addons/mail/static/tests/chatter/web/chatter.test.js

```diff
@@ -577,6 +576,18 @@ test("chatter updating", async () => {
     await contains(".o-mail-Message");
 });
 
+test("chatter message actions appear only after saving the form", async () => {
+    await start();
+    await openFormView("res.partner");
+    await contains(".o-mail-Message");
+    await contains(".o-mail-Message-actions", { count: 0 });
+    await click(".o_form_button_save");
+    await click("button", { text: "Send message" });
+    await insertText(".o-mail-Composer-input", "hey");
+    await click(".o-mail-Composer-send");
+    await contains(".o-mail-Message-actions");
```

Comment:
A bit weird that the "Creating a new record..." doesn't stay in HOOT tests, but that's a minor issue.